### ISSUE TITLE
Additional Equivalent Domains

### ISSUE THREAD

**Issue by Dubz** [bitwarden/server#649]
I have a list of more equivalent domains that could be potentially added to the list. I have added them in my previous vault, but am submitting them here since I have transitioned to a self hosted instance of Bitwarden.

Each line is a different group, and each group is alphabetized.
```
"infinitysrv.com","x10hosting.com","x10premium.com"
"dnsomatic.com","opendns.com","umbrella.com"
"cagreatamerica.com","canadaswonderland.com","carowinds.com","cedarfair.com","cedarpoint.com","dorneypark.com","kingsdominion.com","knotts.com","miadventure.com","schlitterbahn.com","valleyfair.com","visitkingsisland.com","worldsoffun.com"
"ubnt.com","ui.com"
```

Additionally, `office.com` could be added to the list of Microsoft domains. I saw `office365.com` added, but not `office.com`.

I'd make a PR, but I'm not entirely sure what changes need to be made to do so. I figured a simple list and in a CSV format should help expedite it to whoever knows what needs to be done and is willing to do so.

---
Sources:
- x10: x10hosting is free hosting, x10premium is paid (premium) hosting, and infinitysrv.com is used for their servers (ex. cPanel, phpMyAdmin, webmail, etc.). Link to premium site on https://x10hosting.com. cPanel links are viewable to clients logged in.
- OpenDNS is owned by Cisco, as well as Cisco Umbrella. DNS-O-Matic is a DDNS solution from them that integrates with other systems, and uses the same account to login (used with OpenDNS DDNS integration).
- Cedar Fair: https://www.cedarfair.com/our-parks links to all the other parks, and logins work across them (SAML basically).
  - Unalphabetized, in order of links on website: `"cedarfair.com","cagreatamerica.com","canadaswonderland.com","carowinds.com","cedarpoint.com","dorneypark.com","kingsdominion.com","visitkingsisland.com","knotts.com","miadventure.com","valleyfair.com","worldsoffun.com","schlitterbahn.com"` (last two links are the same domain, s I've omitted the duplicate in both copies).
- Ubiquiti: https://ubnt.com redirects to https://www.ui.com. `ui.com` is a new, short domain. Previous domain (and still used in some areas at times) was `ubnt.com`.

**Comment by AKTheKnight** [comment]
infinitysrv.com resolves to 127.0.0.1, are you sure it's in use by them?

**Comment by Dubz** [comment]
Yes, I personally use them. They label their servers on subdomains of infinitysrv.com.
You can visit this URL to see the cPanel error and it stating the hosting provider is x10hosting:
[https://so2.infinitysrv.com](https://so2.infinitysrv.com)
